python-data-version-control/src/evaluate.py

```diff
@@ -0,0 +1,29 @@
+from joblib import load
+import json
+from pathlib import Path
+import numpy as np
+import pandas as pd
+from sklearn.metrics import accuracy_score
```

Comment:
Heh, I do this usually (and automatically, with `isort`) but decided to remove the empty spaces because I thought it would be more in line with the style guide. Sometimes it's tough to find the balance between standard coding advice and the RP style guide. 😅

Reply:
Yeah, I know it can be a little confusing to know where to deviate from PEP 8 and where to follow it. But that's one of the things TR is for! To help you sort all that out!